sherpa/astro/ui/utils.py

```diff
@@ -12401,13 +12409,19 @@ def save_all(self, outfile=None, clobber=False):
         Notes
         -----
 
-        Items which are not saved include:
+        This command will create a series of commands that restores
+        the current Sherpa set up. It does not save the set of commands
+        used. Not all Sherpa settings are saved. Items not fully restored
+        include:
 
-        - user models
+        - data created by calls to `load_arrays`, or changed from the
```

Comment:
Sorry, I'm unclear on what you're asking here. Does save_all still not save data created with a load_arrays call?